src/types/type.ts

```diff
@@ -61,6 +62,7 @@ export interface ThemeProps {
     mobileM: string;
     tablet: string;
     desktop: string;
+    desktopM: string;
```

Comment:
You can write a Record type on `media` and `colors` objects as `Record<string, string>` because these two pairs of values always will be a `string`. type, as I know.